README.md

```diff
@@ -125,28 +125,52 @@ For more examples, see the [examples](examples) directory. For more information
 
 ### Create a model
 
-`ollama create` is used to create a model from a Modelfile.
+Create a model from a Modelfile
+
+```
+ollama create <modelname> -f ./Modelfile
+```
 
 ### Pull a model
 
+Pull a model from the library at [ollama.ai/library](https://ollama.ai/library).
+
 ```
 ollama pull llama2
 ```
 
-> This command can also be used to update a local model. Only the diff will be pulled.
-
 ### Remove a model
 
+Remove the model files from your local system.
+
 ```
 ollama rm llama2
 ```
 
 ### Copy a model
 
+Copy a model on your local system, giving it a new name. The underlying files, such as the model weights, are not duplicated.
```

Comment:
Similar here, I think it's mostly self-explanatory

Reply:
we have gotten a few questions in the discord about whether a copy 'copies' the files which it doesn't.